content/pt-br/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,361 @@
+
+```shell
+kubectl get pods
+```
+
+Irá retornar a seguinte saída:
+```shell
+NAME             READY   STATUS    RESTARTS   AGE
+frontend-hmmj2   1/1     Running   0          9s
+pod1             1/1     Running   0          36s
+pod2             1/1     Running   0          36s
+```
+
+Nesse sentido, um ReplicaSet pode possuir um grupo não-homogêneo de Pods
+## Escrevendo um manifesto ReplicaSet
+
+Como todos os outros objetos de Kubernetes API, um ReplicaSet necessita dos campos `apiVersion`, `kind`, e `metadata`. 
+Para ReplicaSets, o `kind` sempre será um ReplicaSet.
+
+O nome de um objeto ReplicaSet precisa ser [nome de subdomínio de DNS](/pt-br/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
+
+Um ReplicaSet também precisa de uma [seção `.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+### Template de Pod
+
+O `.spec.template` é um [template de pod](/docs/concepts/workloads/pods/#pod-templates) que também necessita de labels configurados. No nosso exemplo `frontend.yaml` nós temos uma label: `tier: frontend`.
+Fique atento para não sobrepor com seletores de outros controllers, para que eles não tentem adquirir esse Pod.
+
+Para o campo de [restart policy](/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy) do template, `.spec.template.spec.restartPolicy`, o único valor permitido é `Always`, que é o padrão.
+
+### Seletor de Pod
+
+O campo `.spec.selector` é um [seletor de labels](/docs/concepts/overview/working-with-objects/labels/). Como discutido [anteriormente](#como-um-replicaset-funciona) esses são os labels usados para identificar Pods em potencial para aquisição. No nosso exemplo `frontend.yaml`, o seletor era:
+
+```yaml
+matchLabels:
+  tier: frontend
+```
+
+No ReplicaSet, `.spec.template.metadata.labels` precisa combinar com `spec.selector`, ou será rejeitado pela API.
+
+{{< note >}}
+Para 2 ReplicaSets definindo o mesmo `.spec.selector` mas diferentes campos de `.spec.template.metadata.labels` e `.spec.template.spec`, cada ReplicaSet ignorará os Pods criados pelo outro ReplicaSet.
+{{< /note >}}
+
+### Replicas
+
+Você pode definir quantos Pods devem executar simultaneamente determinando `.spec.replicas`. O ReplicaSet irá criar/deletar os Pods para igualar à esse número.
+
+Se você não especificar o `.spec.replicas`, seu padrão é 1. 
+
+## Trabalhando com ReplicaSets
+
+### Deletando um ReplicaSet e seus Pods
+
+Para deletar um ReplicaSet e todos os seus Pods, use [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete). O [Garbage collector](/docs/concepts/workloads/controllers/garbage-collection/) automaticamente deleta todos os Pods dependentes por padrão.
+
+Quando usar a API REST ou a biblioteca `client-go`, você precisa definir `propagationPolicy` para `Background` ou `Foreground` na opção -d.
```

Comment:
Suggestion: use code formatting for `-d`, since it refers to a command-line option.